packages/expo-video/src/VideoView.types.ts

```diff
@@ -14,6 +14,13 @@ export declare class VideoPlayer {
    */
   isMuted: boolean;
 
+  /**
+   * Float value between 0 and 1 representing the current volume. This value will be ignored if the player is muted.
```

Comment:
> This value will be ignored

Only setting it will be ignored, but in general it still can return some value. The question is, what do we want to return when it's muted? Should it be `0.0` or just use whatever comes from native?

Reply:
For now I've made it return the native volume value even when it's muted. In my opinion this is the better option, because it gives the user more information. For example people creating their own video controls might want to know the target volume at all times for the volume bar etc. and the `isMuted` property can be checked separately.